client/overview/connection-sucess-notice.tsx

```diff
@@ -60,12 +64,21 @@ const ConnectionSuccessNotice: React.FC = () => {
 				</>
 			) : (
 				<>
-					<h2>
-						{ __(
-							'Congratulations! Your store has been verified.',
-							'woocommerce-payments'
-						) }
-					</h2>
+					{ accountStatus !== 'complete' ? (
```

Comment:
@vladolaru in this case we'll get `Congratulations! Your store has been verified.` only when account is `complete`. That means that both payments and deposits are enabled. 
I think we should change the condition to `accountStatus === 'pending_verirfication'`. In this case when user set up the payments (completed PO) but deposits are not set up yet, they also receive `Congratulations! Your store has been verified.` message. 

WDYT?

Reply:
Hmmm... Have you been able to replicate this? Because when we are doing PO we have [a different copy](https://github.com/Automattic/woocommerce-payments/blob/09b332bdb078bafdd2092b8dfb6d14b9ca8ec96f/client/overview/connection-sucess-notice.tsx#L50-L50) altogether; the condition is `isPoEnabled && ! isPoComplete`.

Reply:
ah! yeah, you're right.